tests/e2e/onboarding.test.ts

```diff
@@ -38,9 +38,18 @@ test('onboarding', async ({ page }) => {
 	await expect(page).toHaveURL(`/signup`)
 
 	const emailTextbox = page.getByRole('textbox', { name: /email/i })
+
+	// Check for the error message
 	await emailTextbox.click()
-	await emailTextbox.fill(onboardingData.email)
+	await emailTextbox.fill('kody@kcd.dev')
+	await page.getByRole('button', { name: /launch/i }).click()
+	await expect(
+		page.getByText(/user already exists with this email/i),
+	).toBeVisible()
```

Comment:
@kentcdodds,

this test work locally but failed on CI
let me know how to fix it

Reply:
Why was this change made?

Reply:
I've just added it to make sure it wok, I can remove it if you don't want it

Reply:
Yes, it's possible: https://github.com/nektos/act

I don't think we want to write the test this way. It has an implicit dependency on the seed script. If we wanted to test this we'd want to make a new user first and then use that one. I don't think that's necessary for this level of testing though. Most of the time E2E should test happy-path stuff. If we want to test this we should go with lower level tests (like unit or integration tests).

Reply:
ok, I will remove it